scripts/mcp_chrome_smoke.py

```
@@ -0,0 +1,169 @@
+        "requested_url": url,
+        "resolved_url": _text_content(href),
+        "title": _text_content(title),
+        "navigation": _text_content(navigation),
+        "screenshot": str(saved),
+        "bytes": saved.stat().st_size,
+    }
+
+
+def main() -> int:
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("--output-dir", default="/tmp/penguin-mcp-chrome-smoke")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/penguin-mcp-chrome-smoke"

(S108)